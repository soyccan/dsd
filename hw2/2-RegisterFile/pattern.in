0_101_010_100_11100101 // 0524e5
1_100_100_000_11010111 // 1440d7
0_101_101_110_11100111 // 0556e7
0_011_010_000_00111010 // 03203a
1_001_011_001_00110110 // 113136
0_001_001_000_00111000 // 011038
1_011_100_011_11101101 // 1343ed
1_011_100_100_11010011 // 1344d3
1_000_101_010_00101110 // 10522e
0_001_111_011_01001111 // 01734f
1_011_010_111_01001010 // 13274a
0_011_110_110_01010110 // 036656
1_001_100_111_01000010 // 114742
0_001_110_110_01111011 // 01667b
0_000_011_101_11001101 // 0035cd
1_010_110_000_00101110 // 12602e
1_001_101_110_00100110 // 115626
1_001_100_001_01010011 // 114153
1_100_011_001_11001101 // 1431cd
0_010_111_100_11001111 // 0274cf
0_100_010_011_00000010 // 042302
1_111_011_010_11110000 // 1732f0
0_011_100_001_11000100 // 0341c4
1_111_101_000_10110011 // 1750b3
1_010_001_110_00111000 // 121638
1_011_000_111_00010000 // 130710
1_110_000_101_00111011 // 16053b
1_001_101_110_01100001 // 115661
0_111_001_000_00001101 // 07100d
1_101_100_001_00111010 // 15413a
0_011_110_111_10000001 // 036781
0_001_101_101_11100110 // 0155e6
0_001_111_011_10000011 // 017383
1_111_010_011_11110010 // 1723f2
0_000_111_011_01100100 // 007364
1_001_000_011_10111000 // 1103b8
1_111_010_011_00110011 // 172333
0_010_011_100_10010101 // 023495
1_000_001_010_11111001 // 1012f9
0_111_101_100_01010100 // 075454
1_011_010_101_00101011 // 13252b
0_100_111_010_01000110 // 047246
0_001_110_100_11100110 // 0164e6
0_011_000_101_01110010 // 030572
0_000_110_101_10110110 // 0065b6
0_101_010_100_10101000 // 0524a8
0_011_001_000_11011001 // 0310d9
1_110_110_101_10110011 // 1665b3
0_010_001_101_10000001 // 021581
0_011_011_000_01110110 // 033076
