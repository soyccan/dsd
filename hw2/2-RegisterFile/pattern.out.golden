bbdd
dd00
eeff
bb00
ccaa
3600
d7cc
d7d7
eebb
00d3
bb00
ffff
d700
ffff
4aee
ff00
eeff
d726
4a53
00cd
2e4a
4a2e
cd53
ee00
53ff
00b3
00ee
ee3b
6100
cd61
3bb3
3a3a
b310
3810
f210
0010
3810
10cd
b838
3acd
383a
3338
3bcd
003a
3b3a
38cd
b800
3b3a
b83a
2b00
